Code/Framework/AzToolsFramework/AzToolsFramework/UI/PropertyEditor/EntityPropertyEditor.hxx

```diff
@@ -559,20 +605,38 @@ namespace AzToolsFramework
 
         QIcon m_emptyIcon;
         QIcon m_clearIcon;
+        QIcon m_dragIcon;
+        QCursor m_dragCursor;
 
         QStandardItem* m_comboItems[StatusItems];
         EntityIdSet m_overrideSelectedEntityIds;
 
         Prefab::PrefabPublicInterface* m_prefabPublicInterface = nullptr;
         bool m_prefabsAreEnabled = false;
 
+                // Reordering row widgets within the RPE.
+        const float MoveStartFadeTime = 0.5f;
```

Comment:
Maybe this could be a constexpr instead?

Reply:
Suggestion: "MoveStartFadeTime" -> "MoveStartFadeSeconds", or maybe even just "MoveFadeSeconds".  I keep reading this as "seconds before starting the move fade", instead of "seconds to fade when starting a move".